bin/somalier_wrapper.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+import os
+import glob
+import csv
+import logging
+import pandas as pd
+
+############################################################################################################
+# Gather arguments
+############################################################################################################
+
+def parse_arguments():
+	argument_parser = argparse.ArgumentParser()
+	argument_parser.add_argument(
+		"--sample_info_csv",
+		help="requires following columns: sample_id, cram_path and (optional) patient_identifier",
+		required=True,
+	)
+	argument_parser.add_argument(
+		"--fasta", help="Path to reference genome", required=True
+	)
+	argument_parser.add_argument("--sites", help="Path to sites file", required=True)
+	argument_parser.add_argument("--prefix", help="Prefix for results", required=True)
```

Comment:
for any of these required arguments that'll be the same every time (e.g. sites file), we add a default so we don't need to give the arguments every time? or if you want it to be more general, we could put those arguments in a config to pass in. Not a necessity though, just a suggestion